workloads/virtual-machines/graphical-and-console-access.md

```diff
@@ -0,0 +1,65 @@
+# Graphical and Serial Console Access
+
+Once a virtual machine is started you are able to connect to the consoles it
+exposes. Usually there are two types of consoles:
+
+* Serial Console
+* Graphical Console \(VNC\)
+
+> Note: You need to have `virtctl` [installed](/installation/?id=client-side-virtctl-deployment) to gain access to the VirtualMachine.
+
+## Accessing the serial console
+
+The serial console of a virtual machine can be accessed by using the
+`console` command:
+
+```bash
+$ virtctl console --kubeconfig=$KUBECONFIG testvm
+```
+
+## Accessing the graphical console \(VNC\)
+
+Accessing the graphical console of a virtual machine is usually done through
+VNC, which requires `remote-viewer`. Once the tool is installed you can
+access the graphical console using:
+
+```bash
+$ virtctl vnc --kubeconfig=$KUBECONFIG testvm
+```
+
+## RBAC Permissions for Console/VNC Access
+
+### Using Default RBAC ClusterRoles
+
+Every KubeVirt installation after version v0.5.1 comes a set of default RBAC
```

Comment:
The 0.5.1-alpha tag is already in the KubeVirt tree. Are these permissions destined for 0.5.1 or the subsequent release?

Reply:
0.5.1-alpha.1 is just a pre-release. It is just a snapshot of a current master state. It does not imply what will be part of 0.5.1 at the end. So this may be very likely be part of 0.5.1.

Reply:
i'll update this once the patch goes in and we're sure

Reply:
Until we are out of alpha it might make sense to simply tag the documentation when we do a release in kubevirt and simply evolve the user-guide without looking back. Once we support sumething it matters much more which version includes what.